src/systemctl/systemctl.c

```diff
@@ -6341,8 +6377,14 @@ static int unit_is_enabled(int argc, char *argv[], void *userdata) {
                         if (STR_IN_SET(s, "enabled", "enabled-runtime", "static", "indirect", "generated"))
                                 enabled = true;
 
-                        if (!arg_quiet)
+                        if (!arg_quiet) {
                                 puts(s);
+                                if (arg_full) {
+                                        r = show_installation_targets(bus, *name);
+                                        if (r < 0)
+                                                return r;
+                                }
+                        }
```

Comment:
looks pretty good. 

One more request though: I figure we should also return useful info in the "client-side" case, i.e. the upper branch of the big if check here... The "service-side" case looks pretty nice now, but we also should suppor the "client-side" case)

(background: "systemctl enable" and friends can operate in two modes: the default is server-side mode, where systemctl just asks pid1 to execute the operation and shows the output of it. This is the mode you are covering nicely now. And then there's the other mode, where the operation is executed client-side in an "offline" way. This mode is mostly used during OS installtion or when --root= is used. We try to keep both modes functionality-wise in sync. What's missing here hence is to call directly into yout install.c code for the client-side branch (i.e. code close to the original code you had).)